docs/src/content/docs/utilities/add-virtual-imports.mdx

```diff
@@ -21,6 +21,13 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
                     addVirtualImports({
                         'virtual:my-integration/config': `export default ${JSON.stringify({ foo: "bar" })}`,
                     })
+                    addVirtualImports([
+                        {
+                            id: "virtual:my-integration/server-only",
+                            content: "export const foo = 'bar'",
+                            context: "server"
+                        }
+                    ])
```

Comment:
There could be a client import here to show that you can have the same import in different contexts